pkg/imgpkg/imagetar/tar_reader.go

```diff
@@ -28,6 +31,98 @@ func (r TarReader) Read() ([]imagedesc.ImageOrIndex, error) {
 	return imagedesc.NewDescribedReader(ids, file).Read(), nil
 }
 
+// GetLayersPresentInFile retrieves all the layers that are present in a tar file
+func (r TarReader) GetLayersPresentInFile() ([]v1.Layer, error) {
```

Comment:
suggestion on naming: GetLayersPresentInFile() => PresentLayers(). InFile is redundant since it's already on TarReader.